src/plugins/intel_gpu/src/plugin/transformations/decompose_reduce_for_false_keepdims.cpp

```diff
@@ -7,27 +7,27 @@
 #include <algorithm>
 #include <cassert>
 #include <memory>
-#include <ngraph/opsets/opset10.hpp>
-#include <ngraph/pattern/op/wrap_type.hpp>
-#include <ngraph/rt_info.hpp>
+#include "openvino/opsets/opset10.hpp"
```

Comment:
IMO, better to avoid opsets and include specific ops

Reply:
agree, fixed that in the plugin code. However, some transformations headers have opset includes which are propagated to the plugin. 
At least these 3:
```
#include "transformations/op_conversions/convert_reduce_to_pooling.hpp"
#include "transformations/op_conversions/convert_reduce_to_reshape.hpp"
#include "low_precision/network_helper.hpp"
```